src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandler.cs

```diff
@@ -56,21 +60,39 @@ public async Task<Result<InvitationsResult>> Handle(GetInvitationsQuery request,
             var invitationIds = invitationForQueryDtos.Select(i => i.Id).ToList();
 
             var invitationsWithIncludes = await GetInvitationsWithIncludesAsync(_context, invitationIds, cancellationToken);
+            var codes = invitationsWithIncludes.SelectMany(x => x.Participants).Where(y => y.FunctionalRoleCode != null).Select(y => y.FunctionalRoleCode).Distinct().ToList();
```

Comment:
Suggestion: Can replace .Where(y => y.FunctionalRoleCode != null) with .Where(y => y.HasRole)  for better readability.